packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/groups/GroupsTests.java

```diff
@@ -31,44 +31,72 @@ public void dataPreparation() throws Exception
     @Test(groups = { TestGroup.REST_API, TestGroup.GROUPS, TestGroup.SANITY })
     @TestRail(section = { TestGroup.REST_API, TestGroup.NODES }, executionType = ExecutionType.SANITY,
             description = "Verify creation, listing, updating and deletion of groups.")
-    public void createListUpdateAndDeleteGroup() throws Exception
-    {
-        String groupName = "ZtestGroup" + UUID.randomUUID().toString();
-        JsonObject groupBody = Json.createObjectBuilder().add("id", groupName).add("displayName", groupName).build();
+    public void createListUpdateAndDeleteGroup() {
```

Comment:
Generally in our Java projects we have opening braces on a new line. The two exceptions I know of are in the Elasticsearch Connector and in lambda expressions.